splunk/packages/kv-editor/src/main/webapp/pages/dashboard/definition.json

```diff
@@ -23,20 +23,37 @@
         "lineLength": 10,
         "width": 240,
         "collection": "example_collection",
+        "labelMap": [
+          {
+            "key": "Score",
+            "label": "ScoreInput"
+          },
+          {
+            "key": "Title",
+            "label": "TitleInput"
+          },
+          {
+            "key": "Year",
+            "label": "YearInput"
+          }
+        ],
         "model": {
           "Score": {
+            "order": 1,
```

Comment:
Why not using an array for model and using the order of the array?

Reply:
yes, that would probably be better in the future, but we will keep it for now like this